public/js/firebase.js

```diff
@@ -133,13 +134,16 @@ const firebaseBot = (function () {
 
   function incrementCaptionCount() {
     let gameID = parseInt(window.location.pathname.substring((window.location.pathname.indexOf("gameID=") + "gameID=".length), (window.location.pathname.indexOf("/", (window.location.pathname.indexOf("gameID=") + "gameID=".length)))));
+    let captionerID = parseInt(window.location.pathname.substring((window.location.pathname.indexOf("playerID=") + "playerID=".length), (window.location.pathname.indexOf("/", (window.location.pathname.indexOf("playerID=") + "playerID=".length)))));
+    let roundNumber = parseInt(window.location.pathname.substring((window.location.pathname.indexOf("roundNumber=") + "roundNumber=".length), (window.location.pathname.indexOf("/", (window.location.pathname.indexOf("roundNumber=") + "roundNumber=".length)))));
     database.ref('games/' + gameID + '/captionCount').once("value").then(function (snapshot) {
       console.log("snapshot value", snapshot.val());
       let newCaptionCount = snapshot.val() + 1;
       console.log("newCaptionCount", newCaptionCount);
       database.ref('games/' + gameID).update({
         captionCount: newCaptionCount
       });
+      location.replace("/phone-vote/gameID=" + gameID + "/playerID=" + captionerID + "/roundNumber=" + roundNumber + "/");
```

Comment:
the database calls to firebase are async as well, `you can .then(location.replace)` off the end of it. In fact probably better in case something unexpected happens.